index.js

```
@@ -58,6 +60,46 @@ app.post("/github_push_webhook/:id", async (req, res) => {
   }
 });
 
+async function post_discord_url_to_db() {
+  try {
```

remove try catch - you're swallowing the error which you should not be doing